server/api/events/index.ts

```diff
@@ -1,8 +1,28 @@
 import { getQuery } from 'h3'
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+
+async function findEventsWithinDistance(city:string, lng:number, lat:number, distanceInKm:number) {
+    const events = await prisma.$queryRaw`
```

Comment:
Did you test it? Copy pasting code is not a solution.

Reply:
i have given a response on slack. pls check.